bec_ipython_client/bec_ipython_client/callbacks/live_table.py

```diff
@@ -303,6 +303,10 @@ def _format_value(self, signal: dict | None, precision: int = 2):
             return "N/A"
         val = signal.get("value")
         if isinstance(val, SupportsFloat) and not isinstance(val, np.ndarray):
+            if precision < 0:
+                # This is to cover the special case when EPICS returns a negative precision.
+                # More info: https://epics.anl.gov/tech-talk/2004/msg00434.php
+                return f"{float(val):.{-precision}g}"
```

Comment:
Can this not just be collapsed to `{abs(precision)}` ?

Reply:
One uses "g", the other one "f" 😉